packages/terra-framework-docs/src/terra-dev-site/test/popup/PopupWithContentWidth.test.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import classNames from 'classnames/bind';
+import Popup from 'terra-popup';
+import styles from './PopupTypeCommon.test.module.scss';
+
+const cx = classNames.bind(styles);
+
+class PopupWithContentWidth extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleButtonClick = this.handleButtonClick.bind(this);
+    this.handleRequestClose = this.handleRequestClose.bind(this);
+    this.setButtonNode = this.setButtonNode.bind(this);
+    this.getButtonNode = this.getButtonNode.bind(this);
+    this.state = { open: false };
+  }
+
+  componentDidMount() {
+    this.forceUpdate();
+  }
+
+  handleButtonClick() {
+    this.setState({ open: true });
+  }
+
+  handleRequestClose() {
+    this.setState({ open: false });
+  }
+
+  setButtonNode(node) {
+    this.buttonNode = node;
+  }
+
+  getButtonNode() {
+    return this.buttonNode;
+  }
+
+  render() {
+    return (
+      <div id="test-popup-area" className={cx('test-popup-area')}>
+        <Popup
+          classNameArrow="test-arrow"
+          classNameContent="test-content"
+          isOpen={this.state.open}
+          targetRef={this.getButtonNode}
+          onRequestClose={this.handleRequestClose}
+          contentWidth="400"
```

Comment:
Can we create a dropdown of height and width dropdown and select value from dropdown on spec file instead of adding hardcoded value..?? 
This test file will be useful to validate future size values as well.  below selector can be used for WDIO to select option from html native-select
```
$('#popupContentHeight > option:nth-child(8)').click(); 
$('#popupContentWidth > option:nth-child(5)').click();
```